tests/utils/autoTester.js

```diff
@@ -257,6 +260,11 @@ setUpTest = function(testCase) {
     if (!isManualMode()) {
         Menu.setIsOptionChecked("Desktop", true);
     }
+
+    // Disable TAA
```

Comment:
I m opposed to that
can we run with TAA on by default instead of off by default

Reply:
The problem is that this causes many tests to fail; and I cannot raise the threshold, making the tests less sensitive.